crypto/src/transaction.rs

```diff
@@ -64,15 +64,9 @@ impl TryFrom<transaction::TransactionBody> for TransactionBody {
             .try_into()
             .map_err(|_| ProtoError::TransactionBodyMalformed)?;
 
-        let expiry_height = proto
-            .expiry_height
-            .try_into()
-            .map_err(|_| ProtoError::TransactionBodyMalformed)?;
+        let expiry_height = proto.expiry_height;
 
-        let chain_id = proto
-            .chain_id
-            .try_into()
```

Comment:
This `try_into` is between the same types, so can be elided

Reply:
woops lol